Makefile

```diff
@@ -298,11 +302,23 @@ fmt-check: fmt
 	@$(call print, "Checking fmt results.")
 	if test -n "$$(git status --porcelain)"; then echo "code not formatted correctly, please run `make fmt` again!"; git status; git diff; exit 1; fi
 
-#? lint: Run static code analysis
-lint: docker-tools
+check-go-version-yaml:
```

Comment:
nit: Needs the `#?` documentation syntax that describes the command for the inline help command.